### ISSUE TITLE
wrepl, sh_srepl, and conf: ip and port should be separately stored in conf.

### ISSUE THREAD

**Issue by Naohiro2g** [Carglglz/upydev#19]
When I was switching back and forth between wrepl via WiFi and sh_srepl via USB-serial, I found a glitch.

```
conf for sh_srepl via USB-serial:
upydev config -t /dev/tty.SLAB_USBtoUART -p pass 

conf for wrepl via WiFi
upydev config -t 192.168.1.32  -p pass 
```

and in the conf file, port and ip is stored bound with the same key "ip". So I have to specify the port or ip inline like follows:

```
upydev sh_srepl -port /dev/tty.SLAB_USBtoUART
upydev wrepl -t 192.168.1.32
```

It could be bettrt if port was stored as "port".

**Comment by Carglglz** [comment]
I'm actually working on this on the develop branch, I've refactored the code base and although it's not finished yet (prototype commands and parallel group mode still need to be refactored) It is pretty stable and with a lot of improvements :) . What you need is both ``upydevice`` and ``upydev`` develop branches, So git clone the repos and install the development branches. I'm also working on the the documentation with readthedocs, so under docs directory you can do ``make html`` and see how its going 👍 . 
This does take some time so the pace is not so fast but for sure it is steady.

About the config file, I think I changed the key to ``addr``, since I added serial and bluetooth low energy compatibility.

So my way to solve this was to make a global group of devices, so one device can only have one address (serial, ip or uuid).

In your case see how to make the global group:
https://github.com/Carglglz/upydev/blob/master/DOCS/HOWTO.md

Again this is improved in the develop branch, so I encourage you to check that version and the [changelog](https://github.com/Carglglz/upydev/blob/develop/changelog.md)

**Comment by Naohiro2g** [comment]
Great news!  I will check the dev version.  I was thinking if you had a plan with readthedocs.

And I'm going to use many MicroPython devices in my classes, so the group system is very good for me.  Thank you for the great tool indeed.

**Comment by Carglglz** [comment]
> Great news! I will check the dev version. I was thinking if you had a plan with readthedocs.

Yes, as I said I've taken some time to refactor the code base and introduce readthedocs to make it easier the maintenance and for others to contribute but I think all the wait and the work will worth it. 👍 

> And I'm going to use many MicroPython devices in my classes, so the group system is very good for me. Thank you for the great tool indeed.

I'm happy to hear that and I'm curious to see how it goes. 👀
